### PR TITLE
Deleted default html

### PR DESCRIPTION
**Pull request by HussainJuned**
Merged the navbar with the main.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,35 +1,18 @@
...
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <nav>
+        <ul>
+          <li>
+            <a href="#">Home</a>
```
Comment (suhel-uis) [inline src/App.tsx:10]: add more menu to the navbar

**Review by suhel-uis — COMMENTED** [review]
Check the comment

File: src/App.tsx (hunk 1 above)
Comment (HussainJuned) [reply src/App.tsx:10]: ok  I will add more.

**Review by suhel-uis — APPROVED** [review]
You can merge the code

**Comment by HussainJuned** [comment]
Merge accepted

**Pull request merged**